_data-prepper/pipelines/configuration/sources/pipeline.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: default
+title: pipeline 
+parent: Sources
+grand_parent: Pipelines
+nav_order: 55
+---
+
+# pipeline
+
+Use the `pipeline` sink to write to another pipeline.
+
+## Configuration options
+
+The `pipeline` sink supports the following configuration options.
+
+| Option | Required | Type   | Description                    |
+|:-------|:---------|:-------|:-------------------------------|
+| name   | Yes      | String | Name of the pipe to read from. |
+
+## Usage
+
+The following example configures a `pipeline` sink that reads from a pipeline named `movies`:
+
+```
+sample-pipeline:
+  source:
+    - pipeline:
+        name: "movies"
+```
```

Comment:
````suggestion
```
{% include copy.html %}
````

Reply:
Looking at the Data Prepper section, none of the yaml snippets have the `yaml` highlighter or copy button added. @vagimeli Could you please put in an issue to fix this?

Reply:
Issue https://github.com/opensearch-project/documentation-website/issues/8223